docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -0,0 +1,71 @@
+
+#### Domain in a Docker image and rolling to an image containing new or unrelated domain directory
+
+The domain is using the "domain in a Docker image" model and the error occurs while rolling to a new Docker image that contains an entirely new or unrelated domain directory.
+
+The problem is that WebLogic cannot support server instances being part of the same WebLogic domain if the server instances do 
+not all share the same domain-specific encryption key and other bootstrapping security details. Additionally, operator introspection
+currently only happens when starting servers following a total shutdown. Therefore, the "boot.properites" generated from
+introspecting the image containing the original domain directory will be invalid when used with a container started with
+the updated Docker image containing the new or unrelated domain directory.
+
+The solution is to either follow the recommended [CI/CD guidelines](https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/) so that the original and new Docker images contain domain directories
+with consistent domain-specific encryption keys and bootstrapping security details or to [perform a total shutdown](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers) of the domain so
+that introspection reoccurs as servers are restarted.
+
+#### Domain in a Docker image after a full domain shutdown and restart
+
+The domain is using the "domain in a Docker image" model and the error occurs while starting servers after a full domain shutdown. 
+
+For this type of failure, it is even more probable that the user name and password specified in the Kubernertes secret referenced by
+`weblogicCredentialsSecret` are wrong. However, there is another common cause. If your development model generates new Docker images
+with new and unrelated domain directories and then tags those images with the same tag, then different Kubernetes worker nodes
+may have different images under the same tag in their local Docker repositories.
+
+The simplest solution is to set `imagePullPolicy` to `Always`; however, the better solution would be to design your development
+pipeline to generate new Docker image tags on every build and to never reuse an existing tag.
+
+#### Domain on a persistent volume and completely replacing the domain directory
+
+The domain is using the "domain on a persistent volume" model and the error occurs while starting servers when the domain directory
+change was made while other servers were still running.
+
+If completely replacing the domain directory then you must [stop all running servers](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers).
+
+Since all servers will already be stopped, there is no requirement that the new contents of the domain directory be related to
```

Comment:
Since -> Because (since – don't use to mean for the reason of; use because. Use since only for matters of time, for example, since the last time the program was run.)